introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb

```diff
@@ -52,25 +52,25 @@
     "\n",
```

Comment:
<p>Formatting: inconsistent usage of space characters after <code>,</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Not sure what you meant here.</p>